packages/wamp/wamp.1.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "Vincent Bernardoff <vb@luminar.eu.org>"
+authors: "Vincent Bernardoff <vb@luminar.eu.org>"
+homepage: "https://github.com/vbmithr/ocaml-wamp"
+bug-reports: "https://github.com/vbmithr/ocaml-wamp/issues"
+license: "ISC"
+dev-repo: "https://github.com/vbmithr/ocaml-wamp.git"
+build: [
+  "ocaml"
+  "pkg/pkg.ml"
+  "build"
+  "--pinned"
+  "%{pinned}%"
+  "--with-yojson"
+  "%{yojson:installed}%"
+  "--with-msgpck"
+  "%{msgpck:installed}%"
+]
+depends: [
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build}
+  "result"
+  "uri"
+]
+depopts: ["yojson" "msgpck"]
+available: [ocaml-version >= "4.01.0"]
```

Comment:
The build's [failing](https://travis-ci.org/ocaml/opam-repository/jobs/203866361) because the code uses attributes, which were introduced in OCaml 4.02:

```
# File "src/wamp.ml", line 10, characters 10-11:
# Error: Syntax error
```

Could you please tighten the `ocaml-version` constraint here?